StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/New Payment Method Screen/AddPaymentMethodViewController.swift

```diff
@@ -41,11 +39,7 @@ class AddPaymentMethodViewController: UIViewController {
             configuration: configuration,
             logAvailability: false
         )
-        if paymentMethodTypes.isEmpty {
-            let errorAnalytic = ErrorAnalytic(event: .unexpectedPaymentSheetError,
-                                              error: Error.paymentMethodTypesEmpty)
-            STPAnalyticsClient.sharedClient.log(analytic: errorAnalytic)
-        }
+
```

Comment:
how come we got rid of this?

Reply:
This is being removed because there is an assert on line 44.  I believe this is more or less a critical error, so we should break loudly.